store/metadata.go

```diff
@@ -217,7 +217,7 @@ func (ms *MetadataStore) getLinkedPeersInfo() (map[string]bool, []metadata.Conta
 			} else {
 				for _, aService := range linkedServices {
 					for _, aContainer := range aService.Containers {
-						if !(aContainer.State == "running" || aContainer.State == "starting") {
+						if !(aContainer.State == "running" || aContainer.State == "starting" || aContainer.State == "stopping") {
```

Comment:
Same comment as in another PR: consider extracting state comparison to a separate call.